requirements.txt

```diff
@@ -1,4 +1,8 @@
 torch
 ConfigSpace
-scikit-learn>=0.22.0,<0.23
+scikit-learn
 pandas
+torchvision
+pytorch-lightning
```

Comment:
Sorry to ask, because I thought we discussed it before and maybe I misunderstood. 

I understood @LMZimmer recommended not having this dependency? 

But again, maybe I missunderstood.

Reply:
dependency on pytorch lightning? As far as I remember we decided to have it as a dependency.

Reply:
Oh, I see. Cool, then I will mark this as resolved